tools/src/py-list-deps.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/env python3.9
```

Comment:
Does this need to be part of this PR? 

Note: I'm not a python guy so not sure of ramifications of this change but based on the other various changes it doesn't seem like this is related to the PR overall.

Reply:
i needed this to run locally, but it does not need to be part of this PR

Reply:
Let's get this removed from the PR.